packages/create-snowpack-app/index.js

```diff
@@ -178,30 +191,22 @@ if (requiredVersion < 10) {
   console.log(chalk.bold.underline(`Quickstart:`));
   console.log(``);
   console.log(`  cd ${targetDirectoryRelative}`);
-  console.log(`  ${useYarn ? "yarn" : "npm"} start`);
+  console.log(`  ${installer} start`);
   console.log(``);
   console.log(chalk.bold.underline(`All Commands:`));
   console.log(``);
   console.log(
     formatCommand(
-      `${useYarn ? "yarn" : "npm"} install`,
+      `${installer} install`,
       "Install your dependencies. (We already ran this one for you!)"
     )
   );
   console.log(
-    formatCommand(
-      `${useYarn ? "yarn" : "npm"} start`,
-      "Start your development server."
-    )
-  );
-  console.log(
-    formatCommand(
-      `${useYarn ? "yarn" : "npm run"} build`,
-      "Build your website for production."
-    )
+    formatCommand(`${installer} start`, "Start your development server.")
   );
   console.log(
-    formatCommand(`${useYarn ? "yarn" : "npm"} test`, "Run your tests.")
+    formatCommand(`${installer} build`, "Build your website for production.")
```

Comment:
`npm build` isn't a thing, unfortunately.

Reply:
Should just be able to add `run` between the installer and `build` to work for all installers.

Reply:
Whoops! Well spotted.